### PR TITLE
Feature/mobile grid

### PR DESCRIPTION
**Pull request by ferrybig**
Add a pc and mobile grid

### CODE REVIEW

File: src/assets/pages/_homepage.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+body {
+	margin: 0;
+}
+@media screen and (min-width: 786px) {
+	body {
+		overflow-y: hidden;
+		overflow-x: visible;
+	}
+}
```
Comment (matthijs186) [inline src/assets/pages/_homepage.scss]: Fix please.

File: src/assets/style.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Modules
+@import 'modules/_vars.scss';
+
+// Partialgus
```
Comment (matthijs186) [inline src/assets/style.scss]: We'll fix this later

**Review by matthijs186 — APPROVED** [review]
Ready to merge.

File: src/assets/pages/_homepage.scss (hunk 1 above)
Comment (peterxqe) [reply src/assets/pages/_homepage.scss]: media query

File: src/assets/pages/_homepage.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+body {
+	margin: 0;
+}
+@media screen and (min-width: 786px) {
```
Comment (matthijs186) [inline src/assets/pages/_homepage.scss]: `min-width` should be `768`

**Review by Drelofs — APPROVED** [review]
Looks good to me! Ready to merge.

**Review by RishiTheGoat — APPROVED** [review]
NICE!

**Review by peterxqe — APPROVED** [review]
top

**Pull request merged**